engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/Warning.scala

```diff
@@ -40,6 +41,16 @@ object Warning {
     override def diagnosticKeys(): Array[Any] = Array()
   }
 
+  case class UnusedImport(
+    override val identifiedLocation: IdentifiedLocation
+  ) extends Warning {
+    override def message(source: IdentifiedLocation => String): String = {
+      "The import is not used"
+    }
+
+    override def diagnosticKeys(): Array[Any] = Array()
+  }
```

Comment:
I have a question, could you show us what a warning for a 'partially used' import looks like?

e.g. if I have
```
from A.B import C, D, E
```
and only `D` is unused, or `D` and `E` are unused, but `C` is used. What it will look like?

I want to be sure that the warning will not suggest that `C` may be unused if it is used.

Reply:
Ah now I noticed we have a separate type for `from` imports: `UnusedSymbolsFromImport` that lists the symbols. Ok that looks great then.

Reply:
> What it will look like?

The particular symbols that are not used from that import will be listed. See the definition of [UnusedSymbolsFromImport warning](https://github.com/enso-org/enso/blob/615bc1ef1e355e958a6b9a080d4bca9da27425fc/engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/Warning.scala#L28-L42).